sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -192,47 +192,52 @@ def _add_argparse_args(cls, parser):
     options = PipelineOptions(['--redefined_flag'])
     self.assertTrue(options.get_all_options()['redefined_flag'])
 
+  # TODO(mariagh): Require unique names only within a test.
```

Comment:
Do you want to make these `TODO(BEAM-1319)`. This notice applies to everyone until that is fixed.